src/aac/plugins/plugin_manager.py

```diff
@@ -11,13 +8,30 @@ def get_plugin_manager() -> PluginManager:
     Returns:
         The plugin manager.
     """
+
+    from aac.plugins import hookspecs, PLUGIN_PROJECT_NAME
```

Comment:
I don't think it's generally a good idea to import within functions. Is there a specific reason we're doing that here, instead of at the top of the file?

Reply:
It was previously due to a circular import, but upon double checking it looks like I must have solved the core issue elsewhere.

Reply:
Nevermind, this issue is that there is a circular dependency loop validator -> plugins -> plugin with validator imported -> validator

```
ImportError: Failed to import test module: test_validator
Traceback (most recent call last):
  File "/home/runner/work/AaC/AaC/.tox/py39/lib/python3.9/site-packages/nose2/plugins/loader/discovery.py", line 201, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/home/runner/work/AaC/AaC/.tox/py39/lib/python3.9/site-packages/nose2/util.py", line 77, in module_from_name
    __import__(name)
  File "/home/runner/work/AaC/AaC/tests/test_validator.py", line 5, in <module>
    from aac.validator import ValidationError, _validate, validation
  File "/home/runner/work/AaC/AaC/src/aac/validator.py", line 12, in <module>
    from aac import plugins, util
  File "/home/runner/work/AaC/AaC/src/aac/plugins/__init__.py", line 8, in <module>
    from aac.plugins import plugin_manager
  File "/home/runner/work/AaC/AaC/src/aac/plugins/plugin_manager.py", line 5, in <module>
    from aac.plugins import (
  File "/home/runner/work/AaC/AaC/src/aac/plugins/gen_json/__init__.py", line 9, in <module>
    from aac.plugins.gen_json.gen_json_impl import print_json
  File "/home/runner/work/AaC/AaC/src/aac/plugins/gen_json/gen_json_impl.py", line 5, in <module>
    from aac.validator import validation
ImportError: cannot import name 'validation' from partially initialized module 'aac.validator' (most likely due to a circular import) (/home/runner/work/AaC/AaC/src/aac/validator.py)
```